### PR TITLE
Remove enzyme and unused testing utils

### PR DESCRIPTION
**Pull request by nsantos16**
## What & Why:

Removed `enzyme` dependency and other unused files related to tests.

## Tasks:
- [x] Tested in different resolutions
- [x] Tested with SSR
- [ ] Tested on Safari browser
- [ ] Design reviewed with design team

### CODE REVIEW

**Review by kevquincke — APPROVED** [review]
Finally! 👏

File: tools/testSetup.js [hunk 1]
```diff
@@ -1,6 +0,0 @@
-import Enzyme from 'enzyme';
```
Comment (JuAnmdq) [inline tools/testSetup.js:1]: We can take advantage of `tools/testSetup.js` that will render at setupFilesAfterEnv to import only once at start `import '@testing-library/jest-dom';` in order to not import in every file common functionalities like `.toBeInTheDocument()`

Comment (nsantos16) [reply tools/testSetup.js:1]: Agree! I will introduce these changes in another PR related to react testing library 😄

Comment (nsantos16) [reply tools/testSetup.js:1]: WDYT about using the next folder structure for that `__tests__/setup/preEnv.js` ?

Comment (JuAnmdq) [reply tools/testSetup.js:1]: In this case, we should move all jest / tl testing related's tools in that folder, is correct?

**Pull request closed without merging**